### PR TITLE
Updated Readme and changed the field name in the cap code.

### PR DESCRIPTION
**Pull request by Sumanshekhar17**
The current cap code has field name set as "temperature" however there is no such field in CDEPS data model component. ESMX Driver only allows those field name which are available in [NUOPCFieldDictionary](https://github.com/ESCOMP/NUOPCFieldDictionary/blob/master/fd.yaml) .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+
+**How does it works?**
+The NUOPC Layer implements a set of generic components that serve as building blocks that can be assembled together in different ways to build a coupled modeling application. In some cases, a generic component can be used as is, and in other cases the generic component must be specialized (customized) for a perticular model or application. Additionaly, the nUOPC Layer defines a set of rules for how components should behave and interact with each other. These technical rules form the backbone of interoperability. NUOPC defines this effective interoperability as the ability of a model component to execute without code changes in a driver that provides the fields that it requires, and to return with informative messages if its input requirements are not met. Any component that follows the NUOPC Layer technical rules are considered as a NUOPC Layer compliant.
+
+Here, we only have NUOPC cap code for DART which is named as `dart_nuopc.F90`. This acts as a model component in ESMF Framework and we are testing this cap code using ESMX application which provides you the executable driver and its easy to test your code in the begening.
```
Comment (hkershaw-brown) [inline README.md:21]: filename is nuopc_dart.F90

```suggestion
Here, we only have NUOPC cap code for DART which is named as `nuopc_dart.F90`. This acts as a model component in ESMF Framework and we are testing this cap code using ESMX application which provides you the executable driver and its easy to test your code in the begening.
```

**Review by hkershaw-brown — APPROVED** [review]
Nice readme, & temperature name change looks good.

Small change of dart_nuopc.F90 -> nuopc_dart.F90 that I've put in as a suggestion on the review.

**Comment by senpaden** [comment]
Some content in here are come from other sources other than us (interns), such as the introduction of the project, and some content on tools and framework like NUOPC and ESMF. Can we include more citation and links to point to where we get the information?

**Comment by senpaden** [comment]
In the first paragraph introducing the project, this was the description of the project before we started. Now that the project is going, did we end up using HPE Cray EX cluster at all?

**Comment by hkershaw-brown** [comment]
> In the first paragraph introducing the project, this was the description of the project before we started. Now that the project is going, did we end up using HPE Cray EX cluster at all?

Derecho is the HPE Cray EX cluster

**Pull request merged**